package.json

```diff
@@ -50,9 +52,13 @@
     "seamless-immutable": "^7.1.2",
     "styled-components": "^2.2.0",
     "uuid": "^3.1.0",
-    "wordcloud": "^1.0.6"
+    "wordcloud": "^1.0.6",
+    "@storybook/react": "^3.2.12",
```

Comment:
should be in devdependencies

Reply:
![image](https://user-images.githubusercontent.com/2098777/31312284-83f61208-abc7-11e7-975e-25c91105b4c3.png)

Reply:
OK, although it's a bit confusing since these dependencies don't end up in the webpack bundle

Reply:
I want fix it, maybe in first time add /* eslint-disable import/no-extraneous-dependencies */ in stories files?